schema/schema.go

```diff
@@ -21,6 +21,12 @@ var svcSwaggerRaw []byte
 //go:embed assets/ui.html
 var rapidocHTML []byte
 
+//go:embed jsonschema/cerbos/policy/v1/Policy.schema.json
+var PolicyJSONSchema []byte
```

Comment:
Because `jsonschema` only reads strings, there's no need to read this file as a `[]byte` and then cast it back to a string.

```suggestion
var PolicyJSONSchema string
```